src/core/control/jobs/BaseExportJob.cpp

```diff
@@ -73,14 +73,28 @@ void BaseExportJob::showFileChooser(std::function<void()> onFileSelected, std::f
         }
     };
 
-    auto popup = xoj::popup::PopupWindowWrapper<xoj::SaveExportDialog>(control->getSettings(), std::move(suggestedPath),
-                                                                       _("Export File"), _("Export"),
-                                                                       std::move(pathValidation), std::move(callback));
+    GtkFileChooserNative* native = gtk_file_chooser_native_new(_("Export File"), GTK_WINDOW(this->control->getWindow()->getWindow()), GTK_FILE_CHOOSER_ACTION_SAVE, _("Export"), _("_Cancel"));
```

Comment:
@bhennion The problem I got is that the PopupWindowWrapper requires a GTKWindow which simply doesn't exist when using gtk_file_chooser_native_new (since the window could be a qt window). So the wrapper cannot support dialogs without a GTKWindow. 

As far as I can see the same with gtk4 [FileDialog](https://docs.gtk.org/gtk4/class.FileDialog.html).

Reply:
@bhennion I tried again using a modified PopupWindowWrapper. I only did change the save dialog and everything seems working. It would be great if you take a look before I write the rest.